### PR TITLE
feat: create ui information of film in detail film page

### PR DESCRIPTION
**Pull request by TonNQ**
Show information of film (name, age rate, category, director, actor, ...) and watch trailer

86ept9dgv

### CODE REVIEW

File: src/components/MovieDetailInfo/constants.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import IconTag from '@/assets/images/ic-tag.svg?react'
+import IconClock from '@/assets/images/ic-clock.svg?react'
+import IconMessage from '@/assets/images/ic-message.svg?react'
+import IconPersonCheck from '@/assets/images/ic-person-check.svg?react'
+import { AGE_RATING } from '@/constants/movie'
+import { formatDate } from '@/utils/datetime'
+
+export const LINE_COUNT = 4
+
+export const GENERAL_INFO = (info) => [
```
Comment (ces-anhlam) [inline src/components/MovieDetailInfo/constants.js:10]: ``` export const GENERAL_INFO = ({category, duration, language = 'No subtitle', ageRate = 'No limit age'} = {}) => {}```

File: src/components/MovieDetailInfo/constants.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  { icon: IconTag, text: info?.category },
+  { icon: IconClock, text: `${info?.duration}'` },
+  { icon: IconMessage, text: `Subtitle: ${info?.language || 'No subtitle'}` },
+  { icon: IconPersonCheck, text: AGE_RATING[info?.ageRate] || 'No limit age' }
+]
+
+export const DESCRIPTION_INFO = (info) => [
```
Comment (ces-anhlam) [inline src/components/MovieDetailInfo/constants.js:17]: same above

File: src/components/VideoPlayer/VideoPlayer.jsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import ReactPlayer from 'react-player'
+import IconClose from '@/assets/images/ic-close.svg'
```
Comment (ces-anhlam) [inline src/components/VideoPlayer/VideoPlayer.jsx:2]: import as component

File: src/styles/transition.css
Comment (ces-anhlam) [inline src/styles/transition.css:1]: Do you use this file?

File: src/components/MovieDetailInfo/constants.js (hunk 1 above)
Comment (TonNQ) [reply src/components/MovieDetailInfo/constants.js]: I fixed

File: src/components/VideoPlayer/VideoPlayer.jsx (hunk 3 above)
Comment (TonNQ) [reply src/components/VideoPlayer/VideoPlayer.jsx]: I fixed

File: src/styles/transition.css
Comment (TonNQ) [reply src/styles/transition.css:1]: Yes. I dont use this file in this commit. I just removed it

**Comment by TonNQ** [comment]
I just added and updated some files that changed API get film following BE (id in query) and put all endpoints into file constants.js

**Review by ces-anhlam — APPROVED** [review]

**Comment by TonNQ** [comment]
I just pushed new code, changed a little in getting data from API (line 22 file MovieDetailPage.jsx: change property 'info' into 'filmInfo') as BE changed that property name

**Review by ces-phatnguyen — APPROVED** [review]

**Pull request merged**